examples/gno.land/r/leon/config/config.gno

```diff
@@ -6,8 +6,10 @@ import (
 )
 
 var (
-	main   std.Address // leon's main address
-	backup std.Address // backup address
+	main            std.Address // leon's main address
+	backup          std.Address // backup address
+	ErrInvalidAddr  = errors.New("config: invalid address")
+	ErrUnauthorized = errors.New("config: unauthorized")
 )
```

Comment:
nit: split this up with at least a new line or separate `var`

Reply:
fixed